scripts/register_missing_combiner_analysis.py

```diff
@@ -24,10 +23,10 @@ def get_sg_ids(vds: str) -> list[str]:
     Returns:
         list[str]: A list of sequencing group IDs.
     """
-    return read_vds(to_path(vds)).variant_data.s.collect()
+    return read_vds(dataset_path(vds)).variant_data.s.collect()
```

Comment:
I'm personally more a fan of passing the full path to the analysis we want to register on the CLI, instead of this helper method taking half a path as an argument.

Reply:
Fair enough, I can change that back.